changes/ee/fix-15844.en.md

```diff
@@ -0,0 +1,7 @@
+Added validation to forbid adding empty usernames to the built-in database authenticator.  Such users cannot be deleted via the HTTP API later, since they mess up the API path.
+
+If you have such an user and wish to delete it, run the following in an EMQX console:
```

Comment:
If not too complex, maybe fix _deletion_ of such a user?

Reply:
Hmm good idea.  Though it's such an unique case that I wonder if it's worth the trouble, since it can be worked around with manual commands.  :sweat_smile: